src/components/CompanyPage.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FaSpinner } from 'react-icons/fa';
+import CompanyTop from './CompanyTop';
+import CompanyBanner from './CompanyBanner';
+import FinanceHeading from './FinanceHeading';
+import FinanceSection from './FinanceSection';
+
+const CompanyPage = function myProject10({ info, symbol }) {
+  const companyInformation = info.responseFinancials;
+  const companyFinanceInfo = info.responseData;
+
+  const financialList = [];
+  if (companyFinanceInfo) {
+    companyFinanceInfo.map((each) => {
+      const {
+        priceEarningsRatioTTM,
+        priceCashFlowRatioTTM,
+        priceBookValueRatioTTM,
+        priceToOperatingCashFlowsRatioTTM,
```

Comment:
- [ ] As stated in the last review kindly add a filter stateless component that enables the user to filter data retrieved on the homepage.

Reply:
I am having trouble addressing this problem with all the pressure of having my project approved.

Reply:
Time is against me